.travis.yml

```diff
@@ -66,6 +66,19 @@ matrix:
         - DESC="package no error"
         - CMD="mvn clean package -Passembly"
 
+    # checkstyle regression no exception (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="checkstyle regression no exception"
+        - CMD1="git clone https://github.com/checkstyle/checkstyle &&"
+        - CMD2="cd checkstyle &&"
+        - CMD3="git checkout -b test-branch &&"
+        - CMD4="git checkout master &&"
+        - CMD5="git reset --hard HEAD^ &&"
```

Comment:
> That means we need to modify one of the module and make commit.

No, like your example pick a specific commit and reset master to the commit before that.
It is the same as us pretending a module was changed.

> What about testing more pairs...

Like I stated, the problem is going to be we don't know what we are testing.
The top 200 commits could be nothing related to modules being changed.